source/code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -66,9 +66,7 @@ namespace PlayFab
         {
             IPlayFabHttpPlugin& http = *PlayFabPluginManager::GetPlugin<IPlayFabHttpPlugin>(PlayFabPluginContract::PlayFab_Transport);
             const auto requestJson = request.ToJson();
-
-            Json::FastWriter writer;
```

Comment:
YES! This will help clear out a lot of issues (may also get rid of the iOS bug https://dev.azure.com/playfab/PlayFab/_workitems/edit/37125 )